packages/manager/src/features/Betas/BetaDetails.tsx

```diff
@@ -0,0 +1,80 @@
+import { AccountBeta } from '@linode/api-v4/lib/account';
+import { Beta } from '@linode/api-v4/lib/betas';
+import Stack from '@mui/material/Stack';
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { Button } from 'src/components/Button/Button';
+import { DateTimeDisplay } from 'src/components/DateTimeDisplay';
+import { Link } from 'src/components/Link';
+import { Typography } from 'src/components/Typography';
+
+interface AccountBetaProps {
+  beta: AccountBeta;
+}
+
+interface BetaProps {
+  beta: Beta;
+}
+
+function BetaDetails(props: BetaProps): React.ReactElement;
+function BetaDetails(props: AccountBetaProps) {
+  const history = useHistory();
+  const {
+    beta: { description, ended, enrolled, id, label, more_info, started },
+  } = props;
+  const startDate = !enrolled ? (
+    <Typography>
+      <strong>Start Date: </strong>
+      <DateTimeDisplay displayTime={false} value={started} />
+    </Typography>
+  ) : null;
+  const endDate = ended ? (
+    <Typography>
+      <strong>End Date: </strong>
+      <DateTimeDisplay displayTime={false} value={ended} />
+    </Typography>
+  ) : null;
+
+  return (
+    <Stack
+      direction="row"
+      justifyContent="space-between"
+      minHeight={66}
+      spacing={2}
+      width="100%"
+    >
+      <Stack minHeight={66} spacing={2} width="100%">
+        <Stack direction="row" justifyContent="space-between">
+          <Typography>
+            <strong>{label}</strong>
+          </Typography>
+          <Stack direction="row" spacing="30px">
+            {startDate}
+            {endDate}
+          </Stack>
+        </Stack>
+        <Typography>{description}</Typography>
+      </Stack>
+      <Stack direction="column" minHeight={66} minWidth="111px" spacing={2}>
+        {enrolled ? null : (
+          <Button
+            onClick={() => {
+              history.push(`/betas/signup`, { betaId: id });
```

Comment:
This doesn't work for me, it just pushes to `/betas/signup` -- changing it to

```suggestion
              history.push(`/betas/${id}/signup`);
```

seems to work though (pushes to `betas/active_closed/signup` for example).